OMCompiler/SimulationRuntime/c/meta/realString.cpp

```diff
@@ -104,3 +110,27 @@ int main()
   }
 }
 #endif
+
+} // end extern "C"
+
+#include <charconv>
+#include <cstdio>
+#include <iterator>
+#include <system_error>
+
+extern "C" {
+
+modelica_string _cpp_realString(modelica_real r)
+{
+  char buffer[32];
+  char buf[32];
+  const auto result = std::to_chars(buf, std::end(buf), r);
+  if (result.ec == std::errc{}) {
+     sfprintf(buffer, "%.*s\n", static_cast<int>(result.ptr - buf), buf);
```

Comment:
Is this really necessary? Shouldn't it be enough to just null-terminate the string?

Reply:
Probably but I didn't care too much about it. There are some issues with the testsuite that baffle me as they make no sense why the fail. I will look into those a bit.